akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ClusterShardingSettings.scala

```diff
@@ -241,6 +243,7 @@ final class ClusterShardingSettings(
       snapshotPluginId,
       stateStoreMode,
       passivateIdleEntityAfter,
+      3.seconds,
```

Comment:
5 as in reference.con?

Reply:
Originally it was hard coded as 3, WDYT?